src/Bridge/Elasticsearch/DataProvider/Paginator.php

```diff
@@ -68,7 +68,15 @@ public function getLastPage(): float
      */
     public function getTotalItems(): float
     {
-        return isset($this->documents['hits']['total']) ? (float) $this->documents['hits']['total'] : 0.;
+        // for elastic search version > 7.0.0
+        if (isset($this->documents['hits']['total']['value'])) {
+            return (float) $this->documents['hits']['total']['value'];
+        }
+        // for elastic search old versions
+        if (isset($this->documents['hits']['total'])) {
```

Comment:
Could you keep the ternary?

Of you can also write `return (float) ($this->documents['hits']['total'] ?? 0.)`;

Reply:
Ok. I write using ternary operator.